src/cgroups/common.rs

```diff
@@ -109,6 +112,16 @@ pub fn create_cgroup_manager<P: Into<PathBuf>>(cgroup_path: P) -> Result<Box<dyn
         }
         (None, Some(cgroup2)) => {
             log::info!("cgroup manager V2 will be used");
+            if systemd_cgroup {
+                if !booted()? {
```

Comment:
This seems to be the only place where the systemd crate is used. You do not need to do this now, but think about if we can check the availability of systemd another way and avoid another dependency. I

Reply:
Agreed.